secure/storage/src/tests/vault.rs

```diff
@@ -100,146 +113,170 @@ fn test_vault_key_value_policies() {
+    noone_store.rotate_key(CRYPTO_KEY).unwrap_err();
+    noone_store.sign(CRYPTO_KEY, &message).unwrap_err();
 
     // Verify reader policy
-    let reader_token = storage.create_token(vec![&reader]).unwrap();
+    let reader_token = storage.create_token(vec![&READER]).unwrap();
     let mut reader_store = VaultStorage::new(dev::test_host(), reader_token, None, None, None);
-    reader_store.export_private_key(key_name).unwrap_err();
+    reader_store.export_private_key(CRYPTO_KEY).unwrap_err();
     assert_eq!(
-        reader_store.get_public_key(key_name).unwrap().public_key,
+        reader_store.get_public_key(CRYPTO_KEY).unwrap().public_key,
         pubkey
     );
-    reader_store.rotate_key(key_name).unwrap_err();
-    reader_store.sign(key_name, &message).unwrap_err();
+    reader_store.rotate_key(CRYPTO_KEY).unwrap_err();
+    reader_store.sign(CRYPTO_KEY, &message).unwrap_err();
 
     // Verify rotater policy
-    let rotater_token = storage.create_token(vec![&rotater]).unwrap();
+    let rotater_token = storage.create_token(vec![&ROTATER]).unwrap();
     let mut rotater_store = VaultStorage::new(dev::test_host(), rotater_token, None, None, None);
-    rotater_store.export_private_key(key_name).unwrap_err();
+    rotater_store.export_private_key(CRYPTO_KEY).unwrap_err();
     assert_eq!(
-        rotater_store.get_public_key(key_name).unwrap().public_key,
+        rotater_store.get_public_key(CRYPTO_KEY).unwrap().public_key,
         pubkey
     );
-    assert_ne!(rotater_store.rotate_key(key_name).unwrap(), pubkey);
-    rotater_store.sign(key_name, &message).unwrap_err();
+    assert_ne!(rotater_store.rotate_key(CRYPTO_KEY).unwrap(), pubkey);
+    rotater_store.sign(CRYPTO_KEY, &message).unwrap_err();
 
-    let new_pubkey = storage.get_public_key(key_name).unwrap().public_key;
+    let new_pubkey = storage.get_public_key(CRYPTO_KEY).unwrap().public_key;
 
     // Verify signer policy
-    let signer_token = storage.create_token(vec![&signer]).unwrap();
+    let signer_token = storage.create_token(vec![&SIGNER]).unwrap();
     let mut signer_store = VaultStorage::new(dev::test_host(), signer_token, None, None, None);
-    signer_store.export_private_key(key_name).unwrap_err();
-    signer_store.get_public_key(key_name).unwrap_err();
-    signer_store.rotate_key(key_name).unwrap_err();
-    let signature = signer_store.sign(key_name, &message).unwrap();
+    signer_store.export_private_key(CRYPTO_KEY).unwrap_err();
+    signer_store.get_public_key(CRYPTO_KEY).unwrap_err();
+    signer_store.rotate_key(CRYPTO_KEY).unwrap_err();
+    let signature = signer_store.sign(CRYPTO_KEY, &message).unwrap();
     signature.verify(&message, &pubkey).unwrap_err();
     signature.verify(&message, &new_pubkey).unwrap();
 }
+
+fn test_vault_tokens() {
+    let mut storage = create_vault_with_namespace(None);
+
+    let partial = Policy::new(vec![Permission::new(
+        Identity::User(WRITER.into()),
```

Comment:
I would just make a distinct identity here...

Reply:
That works too, but I wanted to follow the pattern used by the other tests.. :/